Failure on aarch64-linux [(full log)](https://logs.nix.ci/?key=nixos/nixpkgs.38415&attempt_id=7887ba60-b3a6-4398-bc54-7e1601b2a4ed)

Attempted: rocksdb_lite

<details><summary>Partial log (click to expand)</summary><p>

```
./util/coding.h: In function 'uint64_t rocksdb::DecodeFixed64(const char*)':
./util/coding.h:114:1: warning: control reaches end of non-void function [-Wreturn-type]
 }
 ^
make: *** [Makefile:1835: table/block_based_table_builder.o] Error 1
make: *** [Makefile:1835: db/db_impl_files.o] Error 1
make: *** [Makefile:1835: db/db_impl_readonly.o] Error 1
make: *** [Makefile:1835: db/db_impl_compaction_flush.o] Error 1
builder for '/nix/store/x7w593dyg5cwq0gdfxcfq11bzjhsac8p-rocksdb-5.10.3.drv' failed with exit code 2
[31;1merror:[0m build of '/nix/store/x7w593dyg5cwq0gdfxcfq11bzjhsac8p-rocksdb-5.10.3.drv' failed
```
</p></details>

--- a/pkgs/development/libraries/rocksdb/default.nix
+++ b/pkgs/development/libraries/rocksdb/default.nix
@@ -77,7 +77,7 @@ stdenv.mkDerivation rec {
     homepage = http://rocksdb.org;
     description = "A library that provides an embeddable, persistent key-value store for fast storage";
     license = licenses.bsd3;
-    platforms = platforms.x86_64 ++ platforms.aarch64;
+    platforms = platforms.x86_64;
     maintainers = with maintainers; [ adev wkennington ];
   };
 }

